source/client-side-encryption/tests/unified/createKey-provider-invalid.yml

```diff
@@ -0,0 +1,71 @@
+description: createKey-provider-invalid
+
+schemaVersion: "1.8"
+
+runOnRequirements:
+  - csfle: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+        - commandFailedEvent
```

Comment:
This file doesn't perform any APM assertions.

Reply:
Does the absence of `expectEvents` constitute an assertion that no events occurred? Or should I be including an empty `expectEvents` in these tests to assert that no commands should be executed (my original intent)?